amgut/lib/data_access/ag_data_access.py

```diff
@@ -969,3 +969,303 @@ def is_deposited_ebi(self, barcode):
             sql = "SELECT deposited FROM ag.ag_kit_barcodes WHERE barcode = %s"
             TRN.add(sql, [barcode])
             return TRN.execute_fetchlast()
+
+    # following are DB access functions only used for unit testing:
+
+    def get_random_supplied_kit_id_scanned_unconsented(self):
```

Comment:
that's a great idea. Right now, I don't know how to do that such that still only one DB object is accessing the DB. It would also be help to re-use those functions in labadmin.